test/integration/search/search_test.rb

```diff
@@ -1,4 +1,5 @@
 require "integration_test_helper"
+require "pry-byebug"
```

Comment:
Probably don't want to leave this here - worth sticking in that test helper above though.